tests/features/joinup_subscription/unsubscribe.feature

```diff
@@ -18,14 +18,13 @@ Feature: Unsubscribe from collections
       | Southpark elementary | Eric Cartman | member      |
       | Kenny's house        | Eric Cartman | owner       |
       | Koon's hideout       | Eric Cartman | facilitator |
+    And the following collection content subscriptions:
+      | collection           | user         | subscriptions                     |
+      | Southpark elementary | Eric Cartman | discussion, document, event, news |
+      | Kenny's house        | Eric Cartman | discussion, document, event, news |
+      | Koon's hideout       | Eric Cartman | discussion, document, event, news |
```

Comment:
Users are no longer subscribed automatically when joining a collection now, they have to opt in manually.